src/model_signing/_hashing/file.py

```diff
@@ -140,92 +132,17 @@ def compute(self) -> hashing.Digest:
     @property
     @override
     def digest_size(self) -> int:
-        """The size, in bytes, of the digests produced by the engine."""
```

Comment:
the docstring gets pulled from the parent class?

Reply:
They do. It doesn't make sense to override a docstring for an `@override` method, except when we add something extra.